addons/portal/static/src/scss/portal.scss

```diff
@@ -26,6 +26,11 @@ body {
 }
 
 header {
+    .o_skip_to_content {
+        // We require the z-index property to ensure that the "Skip to Content"
+        // button appears above all other elements and header.
+        z-index: 1000;
```

Comment:
Actually, it's not a good idea to use a fixed value for `z-index`. I think it should at least be `$zindex-modal + 1` :thinking: ? Also, the case where a modal is shown on the page should be considered, for now, the button is just unreachable with a modal on the page...

Reply:
Hello @xO-Tx 
Agree with your point, we changed the z-index value to $zindex-modal + 1, but I'm not clear on which modal case you mean. Can you specify which modal on the page you're referring to? 

CC : @divy-odoo